references/sdk.md

```diff
@@ -36,6 +36,8 @@ add_aggregate_algo(self, data: Union[dict, substra.sdk.schemas.AggregateAlgoSpec
 ```
 
 Create new aggregate algo asset.
+In debug mode, add the following key: `substra.DEBUG_OWNER` to the metadata,
+the value becomes the 'creator' of the objective.
```

Comment:
You've kept `objective` even though this is an aggregate algo. The issue is present pretty much everywhere in the doc.

Reply:
copy paste failure !